examples/hello-world/format-example.cpp

```diff
@@ -0,0 +1,144 @@
+
+#include "internalGroup1.h"
+#include "slang.h"
+#include <algorithm>
+
+#include "includeGroup2.h"
+#include <vector>
+
+#include <vulkan.h>
+
+#ifdef __linux__
+#    include <X11/Xlib.h>
+#else
+#    include <Windows.h>
+#endif
+
+namespace slang
+{
+/// <summary>
+/// Summary of TestClass
+/// </summary>
+class TestClass
+{
+private:
+    int examplePrivateMember; // Trailing comment1
+    int* examplePointerMember = nullptr; // trailing coment2
+    float member4;
+    float member5;
+
+public:
+    TestClass()
+        : examplePrivateMember(0)
+        , examplePointerMember(nullptr)
+        , member4(1.0f)
+        , member5(200.0f)
+    {}
+
+    virtual void emptyFunction() {}
+
+    int shortFunction() { return 0; }
+
+    /// <summary>
```

Comment:
Where we do have parameters explicitly defined it's less verbose to use @param, @return etc.